frame/bags-list/src/lib.rs

```diff
@@ -296,11 +302,11 @@ impl<T: Config<I>, I: 'static> SortedListProvider<T::AccountId> for Pallet<T, I>
 		List::<T, I>::insert(id, score)
 	}
 
-	fn on_update(id: &T::AccountId, new_score: T::Score) {
-		Pallet::<T, I>::do_rebag(id, new_score);
+	fn on_update(id: &T::AccountId, new_score: T::Score) -> Result<(), ListError> {
+		Pallet::<T, I>::do_rebag(id, new_score).map(|_| ())
```

Comment:
Wouldn't it be better if we just let the caller decide what to do with the result rather than mapping it to `()`?

Reply:
Well ok nvm, this is a trait function, better not to mess with it.